src/t8_schemes/t8_default/t8_default_hex/t8_default_hex_cxx.cxx

```diff
@@ -662,4 +662,69 @@ t8_default_scheme_hex_c::~t8_default_scheme_hex_c ()
    * and hence this empty function. */
 }
 
+int
+t8_default_scheme_hex_c::t8_element_pack (const t8_element_t *elements, int count, void *send_buffer, int buffer_size,
+                                          int *position, sc_MPI_Comm comm) const
+{
+  int mpiret;
+  p8est_quadrant_t *quads = (p8est_quadrant_t *) elements;
+  for (int ielem = 0; ielem < count; ielem++) {
+    mpiret = sc_MPI_Pack (&(quads[ielem].x), 1, sc_MPI_INT, send_buffer, buffer_size, position, comm);
+    SC_CHECK_MPI (mpiret);
+    mpiret = sc_MPI_Pack (&quads[ielem].y, 1, sc_MPI_INT, send_buffer, buffer_size, position, comm);
+    SC_CHECK_MPI (mpiret);
+    mpiret = sc_MPI_Pack (&quads[ielem].z, 1, sc_MPI_INT, send_buffer, buffer_size, position, comm);
+    SC_CHECK_MPI (mpiret);
+    mpiret = sc_MPI_Pack (&quads[ielem].level, 1, sc_MPI_INT8_T, send_buffer, buffer_size, position, comm);
+    SC_CHECK_MPI (mpiret);
+  }
+  return 0;
+}
+
+int
+t8_default_scheme_hex_c::t8_element_pack_size (int count, sc_MPI_Comm comm, int *pack_size) const
+{
+  int singlesize = 0;
+  int datasize = 0;
+  int mpiret;
+
```

Comment:
maybe add a small comment here which data is considered (i.e. "a hex has 3 integer (x,y,z) and 1 int8 (level)")

Reply:
I did so at the top of each function